docs/Взаимодействие MS Word и MS Excel/Варианты вставки из Excel.md

```diff
@@ -0,0 +1,61 @@
+﻿# Оглавление Варианты вставки из Excel
+
+## Введение
+
+Microsoft Excel - программа для работы с электронными таблицами, созданная корпорацией Microsoft для Microsoft Windows, Windows NT и Mac OS, а также Android, iOS и Windows Phone.
+
+Она предоставляет возможности экономико-статистических расчетов, графические инструменты, язык макропрограммирования потоков данных Power Query и, за исключением Excel 2008 под Mac OS X, язык макропрограммирования VBA.
+
+В этой статье будет рассмотрены 6 вариантов вставки из Excel, чем отличается копирование-вставка ячеек от копирования-вставки столбцов, что делать, если вставляются пустые ячейки при копировании столбцов.
+
+## 6 видов вставки
+
+В Excel существует несколько вариантов вставки данных:
+
+1. Вставка значений: при этом вставляются только значения из выделенных ячеек или диапазона, без форматирования и формул.
+
+2. Вставка форматов: при этом копируется только форматирование выделенных ячеек или диапазона, без самих значений или формул.
+
+3. Вставка формул: при этом копируются как значения, так и формулы из выделенных ячеек или диапазона.
+
+4. Вставить ссылки: это позволяет создавать ссылки на исходные ячейки или диапазоны, чтобы изменения в оригинальных данных автоматически отражались в вставленных данных.
+
+5. Вставка специальных значений: под этим подразумевается вставка значений с определенными атрибутами, такими как даты, числа или текст.
+
+6. Вставка объектов: это позволяет вставлять объекты, такие как изображения, графики или диаграммы.
+
+
+
```

Comment:
[ch-md-error-separate-blocks](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-separate-blocks)